python/cuml/test/test_array.py

```diff
@@ -27,8 +28,8 @@
 from numba import cuda
 from cudf.core.buffer import Buffer
 from cuml.common.array import CumlArray
-from cuml.common.memory_utils import _get_size_from_shape
-from rmm import DeviceBuffer
+from cuml.common.memory_utils import _get_size_from_shape, _strides_to_order
+import rmm
```

Comment:
why the change from just importing `DeviceBuffer`? Nothing else from rmm python is being used, is it? If change is necessary, please put this `import` above

Reply:
Good catch. I was planning on using something else from `rmm` but scrapped it and forgot to revert the input. Fixed